components/web/php/php-7_3/Makefile

```diff
@@ -113,6 +113,7 @@ CONFIGURE_OPTIONS   +=	--with-imap-ssl
 CONFIGURE_OPTIONS  +=	--with-mysqli=shared,$(MYSQL_BINDIR)/mysql_config
 CONFIGURE_OPTIONS  +=	--with-pdo-mysql=shared,$(MYSQL_BINDIR)/mysql_config
 CONFIGURE_OPTIONS  +=	--with-pgsql=shared,$(PG_BINDIR)
+CONFIGURE_OPTIONS  +=	--with-pdo-pgsql=shared
```

Comment:
pgsql pdo module is compiled using arbitrary postgresql version installed on the system, not Postgresql 9.4

Reply:
Whoops, fixed.
I'm not sure if it's necessary or if php's configure script can use the PG_CONFIG in the CONFIGURE_ENV. It definitely found and compiled against the correct version on my machine, but better safe than sorry.